.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: 'Validate'
+
+on:
+  push:
+    branches:
+      - main
+      - polaris-release
+  pull_request:
+
+jobs:
+  build:
+    name: 'Build, Lint, & Test: node-${{ matrix.node_version }}'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node_version: ['14.17.0', '16.13.0']
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+
+      - name: Set node version to ${{ matrix.node_version }}
+        uses: actions/setup-node@v3
+        with:
+          node_version: ${{ matrix.node_version }}
+          cache: yarn
+
+      - name: Restore cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            *.tsbuildinfo
```

Comment:
```suggestion
            **/*.tsbuildinfo
```

I believe we need to recursively cache these `tsbuildinfo` files:
![image](https://user-images.githubusercontent.com/32409546/166606031-7d791d63-efb1-49d1-bb81-8096dc8133f6.png)

Reply:
Good catch, yes. Ty!